react-challenge.html

```diff
@@ -43,17 +44,24 @@
         render(){
           return(
             <div>
-              <MoodMeter
-                moodOptions={ this.state.moods[this.state.currentMood] }
+             <Header/>
```

Comment:
✔️

Reply:
Thanks again, Rachael. I was not able to figure out how to make the Mood Options iterate. Can you point me in the right direction?